tests/target/configs-where_single_line.rs

```diff
@@ -1,3 +1,12 @@
+// rustfmt-where_single_line: true
+// Where style
+
+
+fn lorem_two_items<Ipsum, Dolor, Sit, Amet>() -> T 
+where Ipsum: Eq, Lorem: Eq {
+    // body
+}
+
 fn lorem<Ipsum, Dolor, Sit, Amet>() -> T 
```

Comment:
Could you also add a test for a single-item where clause where the rest of the signature covers multiple lines? (That should not get single-lined).

Reply:
Could help me with this one? We already have one single-item where clause. What do you mean by "rest of the signature"?

Reply:
If we would put the function signature on multiple lines (e.g., if we use mutliple lines for arguments), then we should always use the multiple line form for the where clause.